charts/oaas-observability/Chart.yaml

```
@@ -5,7 +5,7 @@ description: A Helm chart to deploy obeservability stack on Kubernetes
 home: https://github.com/neticdk/k8s-oaas-observability
 sources:
   - https://github.com/neticdk/k8s-oaas-observability
-version: 2.2.0
+version: 2.3.0
```

I think this would be ok to just qualify as patch version.

```suggestion
version: 2.2.1
```